db/demo_seeds/legislation_processes.rb

```diff
@@ -1,480 +1,861 @@
+      body: "Bike Only Streets is already a working concept" \
+            " in cities like Valencia, Spain, and Montreal, Canada",
+      user_id: 5,
+      created_at: 6.days.ago
+    )
+    up_voters = users.sample(2)
+    up_voters.each do |voter|
+      comment.vote_by(voter: voter, vote: true)
+    end
+
+    comment = question.comments.create!(
+      body: "Districts should become mobility centers in themselves " \
+            "because local communities will benefit the most from that",
+      user_id: 7,
+      created_at: 6.days.ago
+    )
+    up_voters = users.sample(1)
+    up_voters.each do |voter|
+      comment.vote_by(voter: voter, vote: true)
+    end
+    comment.update!(updated_at: 3.days.ago)
+
+    #
+    # Draft
+    #
+
+    body = File.read(Rails.root.join("db", "demo_seeds", "documents", "legislation", "processes",
+                                     "draft-mobility.html"))
+    draft = process.draft_versions.create!(title: "Draft of the Strategic Mobility Plan 2025-2035",
+                                           body: body,
+                                           created_at: 10.days.ago,
+                                           updated_at: 10.days.ago,
+                                           status: "published")
+
+    draft.annotations.create!(
+      quote: "STRATEGIC MOBILITY PLAN 2025-2035",
+      ranges: [{
+        "start" => "/p[1]",
+        "startOffset" => 0,
+        "end" => "/p[1]",
+        "endOffset" => 36
+      }],
+      text: "Maybe we can give it a better sounding name, like Mobility and " \
+            "Transport for All, Sustainable and Efficient. How about that?",
+      author_id: 3,
+      range_start: "/p[1]",
+      range_start_offset: 0,
+      range_end: "/p[1]",
+      range_end_offset: 36,
+      context: "<span class=annotator>STRATEGIC MOBILITY PLAN 2025-2035</span>",
+      created_at: 9.days.ago,
+      updated_at: 9.days.ago
+    )
+    comment = Comment.last
+    up_voters = users.sample(1)
+    up_voters.each do |voter|
+      comment.vote_by(voter: voter, vote: true)
+    end
+
+    draft.annotations.create!(
+      quote: "In five years, zero-emission mobility",
+      ranges: [{
+        "start" => "/p[3]",
+        "startOffset" => 0,
+        "end" => "/p[3]",
+        "endOffset" => 40
+      }],
+      text: "I think this is a very good ambition but I think the texts " \
+            "lacks sufficient proposals to make this a reality",
+      author_id: 3,
+      range_start: "/p[3]",
+      range_start_offset: 0,
+      range_end: "/p[3]",
+      range_end_offset: 40,
+      context: "<span class=annotator>In five years, zero-emission mobility</span>",
+      created_at: 9.days.ago,
+      updated_at: 9.days.ago
+    )
+    draft.annotations.create!(
+      quote: "Eurocities manifesto",
+      ranges: [{
+        "start" => "/p[5]",
+        "startOffset" => 20,
+        "end" => "/p[5]",
+        "endOffset" => 41
+      }],
```

Comment:
Just a quick note about this annotation range.

It looks like we have a small problem when the selected text starts with a space. This is not a new problem. It already happened before.

But since this is demo data, maybe we can avoid this case here.

You can fix it just by changing the offset to:


```suggestion
      ranges: [{
        "start" => "/p[5]",
        "startOffset" => 21,
        "end" => "/p[5]",
        "endOffset" => 41
      }],
```